GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -1,72 +1,267 @@
+                                       </div>
+                                   }
+                                   else
+                                   {
+                                       <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-5">
+                                           @viewModel.GetRequiredToReportOrNotText()
+                                       </p>
+                                   }
+                               </div>
+                               
+                               <strong class="govuk-tag @viewModel.GetReportTagClassName()">
+                                   @viewModel.GetReportTagText().ToUpper()
+                               </strong>
+                               
+                               <div>
+                                    @{
+                                        string description = viewModel.GetReportTagDescription();
+                                        string modifiedDateText = viewModel.GetModifiedDateText();
+                                        string modifiedDateTextClass = description != null ? "" : "govuk-!-margin-top-4";
+        
+                                        if (description != null)
+                                        {
+                                            <div class="govuk-!-margin-top-1">@description</div>
+                                        }
+        
+                                        if (modifiedDateText != null)
+                                        {
+                                            <div class="@modifiedDateTextClass">@modifiedDateText</div>
+                                        }
+                                    }
+                               </div>
+                               <br/>
+                               
+                               <div class="govuk-!-margin-bottom-5">
+                                    <a loadtest-id="create-report-@viewModel.ReportingYear"
+                                       href="@Url.Action("ReportOverview", "ReportOverview", new {encryptedOrganisationId, reportingYear = viewModel.ReportingYear, canTriggerLateSubmissionWarning = true})">
+                                        @viewModel.GetReportLinkText()
+                                    </a>
+                               </div>
+                           </text>
+                },
             }
-        }
-    </tbody>
-</table>
-</div>
\ No newline at end of file
+        };
+        rowList.Add(newRow);
+    }
+}
+
+<h2 class="govuk-heading-l gpg-govuk-hideOnMobile">Manage reports</h2>
+<h2 class="govuk-heading-l gpg-govuk-hideOnDesktop govuk-!-margin-bottom-0">Manage reports</h2>
+@{
+    var missingReports = desktopMissingReportingPeriodRows.Count > 0;
+    var mobileReportingPeriodsClasses = missingReports ? "gpg-govuk-hideOnDesktop govuk-!-margin-bottom-9" : "gpg-govuk-hideOnDesktop govuk-!-margin-bottom-1";

```

Comment:
Because there is a link to all reports after the tables, we need to make sure that the final table has a small margin to keep the all reports close to the end of the table. 
Put another way, if there's missing reports this table won't be the last one so we need a big margin.